scripts/deploy.sh

```diff
@@ -192,15 +192,24 @@ EOF
     exit 1
   fi
 
-  src_path=$(realpath "$simulator_path/src/aoai-simulated-api")
+  image_name="aoai-simulated-api:latest"
+  set +e
+  existing_image=$(az acr repository show --name $acr_name --image $image_name 2>&1)
+  set -e
 
-  # create a tik_token_cache folder to avoid failure in the build
-  mkdir -p "$src_path/tiktoken_cache"
+  if echo "$existing_image" | jq . > /dev/null 2>&1; then
```

Comment:
I tested this and I'm not seeing it skip the build step. I manually ran the `az acr repository show` command and (despite the docs indicating otherwise) it doesn't seem to find my image if the tag is included.

Also, I'd love a way to force the build (without having to figure out deleting the remote image). Maybe add a `FORCE_SIMULATOR_BUILD` var?

Reply:
Interesting, keeping the tag in worked for me! I'll update and add the ability to force the build.